crates/core_simd/src/vectors_f32.rs

```diff
@@ -1,23 +1,29 @@
-define_vector! {
+define_float_vector! {
     /// Vector of two `f32` values
     struct f32x2([f32; 2]);
+    bits crate::u32x2;
```

Comment:
Could use union if you want it to remain rusty

Reply:
There's no union involved though, so this would be a little confusing I think. `bits` isn't valid syntax, but seems more obvious that it's involved with the `from_bits` methods.

I also wouldn't be surprised if we need more things here over time, and we'll run out of relevant rust keywords quickly.

Reply:
Yeah I know, there's no array involved either, really.  I'm fine with it as is, just spitballing.